### DISCUSSION TITLE
still getting 504 error

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#255] [Q&A] [answered]
i delete everything and update with new fork here you cand see [Vercel](https://onedrive-lake.vercel.app/) and here is [Site.json](https://github.com/therockstarind/onedrive/blob/main/config/site.json) please look and tell what worng i did. please

**Comment by spencerwooo** [comment, marked as answer]
try the latest build with error handling, current information doesn't help much for me helping you debug your problem

**Reply by hirockstar** [reply]
i did it again brother still same check here [Link](https://onedrive-steel.vercel.app/)

**Reply by spencerwooo** [reply]
sorry but i don't see anything wrong with your configs, can you do an `npm run dev` locally and check for debug logs?

**Reply by hirockstar** [reply]
do You  have teamviewer?

**Reply by spencerwooo** [reply]
sorry that is out of my league, i cannot debug for you, the project runs perfectly for everybody else.

**Reply by hirockstar** [reply]
bro check for once if you can please.

**Reply by hirockstar** [reply]
im on step 3  what's after that, im seeing this button (Do not pretend to be the site owner)

**Reply by spencerwooo** [reply]
that means you haven't defined (or wrongly defined) `userPrincipalName` in `site.json`, which is used to verify that you are actually you when doing oauth. see https://github.com/spencerwooo/onedrive-vercel-index/discussions/241

**Reply by spencerwooo** [reply]
that `userPrincipalName` is usually the email u used to login into the microsoft oauth process in step 2

**Reply by hirockstar** [reply]
working now thanks brother btw any plans for google drive like this index.

**Reply by spencerwooo** [reply]
probably not gonna extend support for other drives, as it would be too much effort for me as the single maintainer of the project. sry
